noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -1,23 +1,8 @@
-use dep::aztec::{
-    context::PrivateContext,
-    keys::getters::{get_nsk_app, get_public_keys},
-    macros::notes::note,
-    note::{
-        note_interface::NullifiableNote, retrieved_note::RetrievedNote,
-        utils::compute_note_hash_for_nullify,
-    },
-    oracle::random::random,
-    protocol_types::{
-        address::AztecAddress, constants::GENERATOR_INDEX__NOTE_NULLIFIER,
-        hash::poseidon2_hash_with_separator,
-    },
-};
+use aztec::{macros::notes::note, oracle::random::random, protocol_types::address::AztecAddress};
 
 // TODO(#12008): Remove the need for the manual import of `Packable` trait here. This is a bug in macros.
 use aztec::protocol_types::traits::Packable;
 
-pub(crate) global VALUE_NOTE_LEN: u32 = 3; // 3 plus a header.
```

Comment:
Sneaked this in (😝) - we relied on VALUE_NOTE_LEN being exported which is an anti-pattern.